src/renderer/api/endpoints/helm-releases.api.ts

```diff
@@ -57,6 +57,7 @@ export interface IReleaseRevision {
   updated: string;
   status: string;
   chart: string;
+  app_version: string;
```

Comment:
We use camel case for interface field names
```suggestion
  appVersion: string;
```

Reply:
@Nokel81 I tried `appVersion` but it didn't work. I checked API response and in response they used app_version
<img width="617" alt="Screen Shot 2021-02-04 at 7 45 22 PM" src="https://user-images.githubusercontent.com/38247153/106918012-43dc6b80-6722-11eb-9d22-09f83ad71ad5.png">

Reply:
Oh I didn't realize that this interface was representing the data gotten directly.